core/README.md

```diff
@@ -1,22 +1,149 @@
+  <source media="(prefers-color-scheme: light)" srcset="../assets/Product_Icons_light/automations-light.svg" width="14.0%">
+  <img alt="Weights and Biases Prompts" src="">
+</picture>
+</p>
+
+See the [W&B Developer Guide](https://docs.wandb.ai/?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=documentation) and [API Reference Guide](https://docs.wandb.ai/ref?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=documentation) for a full technical description of the W&B platform.
+
+# `wandb-core`: A new backend for the W&B SDK
 
 ## Introduction
 
 `wandb-core` is a new and improved backend for the W&B SDK that is more performant, versatile, and robust.
 
-## Getting started
+**Upgrade now and experience:**
+
+* 🚀 **Logging performance:** Experience up to 88% performance improvements when logging with multiple processes.
+* 🤏 **Reduced resource consumption:** Lower memory footprint allows you to run more experiments on your machines.
+* 📊 **Improved Table logging:** Experience up to 40% faster table logging performance! 
+* ⚡ **Faster startups & shutdowns:** Enjoy up to 36% faster startup and shutdown times.  
+* ⏩ **Enhanced artifact handling:** Experience up to 33% faster Artifact uploads speed and up to 27% faster Artifact retrieval!
+* 🌐 **Faster offline sync:** Keep your long-running experiments synced effortlessly with improved offline speeds.
+
+See our (benchmark analysis)[https://github.com/wandb/wandb/blob/main/tools/bench/bench.md] for more information on performance improvements.
+
+## Quickstart
+Get started with wandb-core in four steps:
+
+1. First, sign up for a [free W&B account](https://wandb.ai/login?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=quickstart).
+
+2. Second, install the W&B SDK with [pip](https://pip.pypa.io/en/stable/). Navigate to your terminal and type the following command:
+
+```bash
+pip install wandb
+```
+***Note: ensure you have `wandb>=0.17.4`.***
+
+3. Third, log into W&B:
+
+```python
+wandb.login()
+```
+
+4. Use the example code snippet below as a template to integrate W&B to your Python script:
 
 To start using `wandb-core`, add `wandb.require("core")` to your script after importing `wandb`:
 
+<p align='center'>
+<a href="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/intro/Intro_to_Weights_%26_Biases.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" /></a>
+</p>
+
 ```python
 import wandb
 
+# Add requirement for wandb core
 wandb.require("core")
+
+# Start a W&B Run with wandb.init
+run = wandb.init(project="my_first_project")
+
+# Save model inputs and hyperparameters in a wandb.config object
+config = run.config
+config.learning_rate = 0.01
+
+# Model training code here ...
+
+# Log metrics over time to visualize performance with wandb.log
+for i in range(10):
+    run.log({"loss": loss})
 ```
 
-Note: ensure you have `wandb>=0.17.0`.
 
-### Platform compatibility
+
+
+## Compatibility
```

Comment:
we wanted to make the table collapsible

Reply:
I looked in to making this collapsable and its easy, but the issue is how GIthub renders the markdown footnotes. The footnotes are rendered outside of the collapsable menu. I think the footnotes help with readability, so opted not to make this collapsable.

Reply:
Ah good to know. Yeah in that case I think it’s better to keep as is

Reply:
Actually, I think I figured it out. Updated!